### PR TITLE
adding some values to config.yaml file like namespaces etc

### PR DESCRIPTION
**Pull request by teiling88**
first of all i love your cli tools <3 

during my usage i have collected some small improvements in my daily work: 
- added a PluginConfig block in the config.yaml to configure author, link, namepsace etc
- added/corrected some phpdoc comments
- improve some code formatting (blank lines etc)

i hope guys you like it ;-)

### CODE REVIEW

**Comment by bcremer** [comment]
@dnoegel looks good to me. care to review?

File: src/Extensions/Shopware/PluginCreator/template/Bootstrap.tpl [hunk 1]
```diff
@@ -42,7 +42,7 @@ class Shopware_Plugins_<?= $configuration->namespace; ?>_<?= $configuration->nam
         $tool = new \Doctrine\ORM\Tools\SchemaTool($em);
 
         $classes = array(
-            $em->getClassMetadata('Shopware\CustomModels\<?= $configuration->name; ?>\<?= $names->camelCaseModel; ?>')
+            $em->getClassMetadata('<?= $configuration->pluginConfig['namespace']; ?>\CustomModels\<?= $configuration->name; ?>\<?= $names->camelCaseModel; ?>')
```
Comment (dnoegel) [inline src/Extensions/Shopware/PluginCreator/template/Bootstrap.tpl]: The "registerCustomModels" call will register models in the "Shopware\CustomModels" namespace. Even if it is technically possible to have models in another namespace (I guess), I think you change set does not reflect this by e.g. feeding the AnnotationReader with this model namespace, if I am not mistaken. So I would suggest to keep "Shopware\CustomModels", even if the rest of your namespace are e.g. "\BestIT\SomeThing\Fancy"

File: src/Extensions/Shopware/PluginCreator/template/Models/Model.tpl [hunk 2]
```diff
@@ -1,9 +1,10 @@
 <?= $configuration->phpFileHeader; ?>
 <?= $configuration->licenseHeader; ?>
 
-namespace   Shopware\CustomModels\<?= $configuration->name; ?>;
-use         Shopware\Components\Model\ModelEntity,
-    Doctrine\ORM\Mapping AS ORM;
+namespace <?= $configuration->pluginConfig['namespace']; ?>\CustomModels\<?= $configuration->name; ?>;
```
Comment (dnoegel) [inline src/Extensions/Shopware/PluginCreator/template/Models/Model.tpl]: See comment above, regarding "registerCustomModels"

Comment (dnoegel) [inline src/Extensions/Shopware/PluginCreator/template/Models/Model.tpl]: :+1: Thank you, looks great. There might be an issue with the CustomModels, however. Can you have a look at it?

**Comment by teiling88** [comment]
@dnoegel i fixed it. i will squash the commits after you reviewed it once again :)

**Comment by dnoegel** [comment]
Thanks @teiling88 !

**Pull request merged**